up-linux-streamer-plugin/vsomeip-configs/mE_client.json

```diff
@@ -0,0 +1,9 @@
+{
+  "applications" :
```

Comment:
Do we have a description of how to do the configuration anywhere?

Reply:
There is some level of assumed knowledge here.

I have to make modifications to the vsomeip transport that would invalidate any docs I would write here, so I'll make an issue #44

Reply:
Going to mark as resolved to be tackled in #44